### PR TITLE
migration changes

### PR DESCRIPTION
**Pull request by jstelnik**

### CODE REVIEW

File: server/prisma/migrations/20230718175518_export_type_2/migration.sql [hunk 1]
```diff
@@ -5,6 +5,6 @@
 
 */
 -- AlterTable
-ALTER TABLE "Shift" DROP COLUMN "foodTaken",
-ADD COLUMN     "damagedFoodTaken" DOUBLE PRECISION NOT NULL DEFAULT 0,
-ADD COLUMN     "regularFoodTaken" DOUBLE PRECISION NOT NULL DEFAULT 0;
+ALTER TABLE "Shift" DROP COLUMN IF NOT EXISTS "foodTaken",
```
Comment (ChrisWilhelm) [inline server/prisma/migrations/20230718175518_export_type_2/migration.sql:8]: uh, wouldn't this be drop column if exists. The other way doesn't make any sense

Comment (ChrisWilhelm) [reply server/prisma/migrations/20230718175518_export_type_2/migration.sql:8]: or did prisma make this for you

**Review by ChrisWilhelm — CHANGES_REQUESTED** [review]

File: server/prisma/migrations/20230718175518_export_type_2/migration.sql (hunk 1 above)
Comment (jstelnik) [reply server/prisma/migrations/20230718175518_export_type_2/migration.sql:8]: oh yeah i'm dumb

File: server/prisma/migrations/20230718181144_form_interaction/migration.sql [hunk 2]
```diff
@@ -5,6 +5,6 @@
 
 */
 -- AlterTable
-ALTER TABLE "Shift" DROP COLUMN "foodTaken",
-ADD COLUMN     "damagedFoodTaken" DOUBLE PRECISION NOT NULL DEFAULT 0,
-ADD COLUMN     "regularFoodTaken" DOUBLE PRECISION NOT NULL DEFAULT 0;
+ALTER TABLE "Shift" DROP COLUMN IF NOT EXISTS "foodTaken",
```
Comment (ChrisWilhelm) [inline server/prisma/migrations/20230718181144_form_interaction/migration.sql:8]: buddy

**Review by ChrisWilhelm — CHANGES_REQUESTED** [review]

**Review by ChrisWilhelm — APPROVED** [review]

File: server/prisma/migrations/20230718181144_form_interaction/migration.sql (hunk 2 above)
Comment (jstelnik) [reply server/prisma/migrations/20230718181144_form_interaction/migration.sql]: yeah yeah

**Pull request merged**